### PR TITLE
Update decide check to use the correct role

### PR DESCRIPTION
**Pull request by ellie**
## Description

We now have a role just for decide, so use it! Ensures we don't check more than we need

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: charts/posthog/Chart.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 30.5.1
+version: 30.5.2
```
Comment (hazzadous) [inline charts/posthog/Chart.yaml:14]: The annoying thing about this is that we are not backwards compatible with older docker images. In this case I _think_ we'll require all dependencies. So would make sense to ensure the upgrade is not rolled out automatically, maybe a minor or major version.

I haven't found a great way to handle this, aside from maybe removing the `role` abstraction and rather allowing specifically the dependencies that should be checked.

I've not got a good answer here 🤷

Comment (xvello) [reply charts/posthog/Chart.yaml:14]: That decide pool is not enabled by default, and requires two values to be set before receiving traffic, so I think it's fine to move forward.
We can add a comment to the `decide.enabled` value that it requires posthog 1.44.0 (we'll do this one before switching to continuous releases, right?)

**Review by xvello — APPROVED** [review]

File: charts/posthog/Chart.yaml (hunk 1 above)
Comment (hazzadous) [reply charts/posthog/Chart.yaml:14]: Hah yeah forgot it was behind a flag. Yeah adding a note to the release doc sounds like a plan 👍

**Comment by xvello** [comment]
Took the liberty of rebasing and adding the comment about required PH version to the values. Ready to merge!

**Review by hazzadous — APPROVED** [review]

**Pull request merged**